packages/nextjs/app/debug/_components/contract/ContractUI.tsx

```diff
@@ -61,6 +61,7 @@ export const ContractUI = ({
               <div className="flex flex-col gap-1">
                 <span className="font-bold">{contractName}</span>
                 <Address address={deployedContractData.address} />
+                <Address address={deployedContractData.classHash} size="xs" />
```

Comment:
the classhash shouldnt use the Address type because it is not an Address. please refer to [Starknet Documentation](https://docs.starknet.io/architecture-and-concepts/smart-contracts/class-hash/)

Tip: 
-  the class hash component should be a component that looks similar to Address but it's less complex , for example it doesnt have `StarkProfile`

in order to make a tag , we can put `Class Hash :`  in front of the Class Hash component. 

> should be clickable and send to starkscan


should be sent to this link format: https://sepolia.starkscan.co/class/{CLASS_HASH}, currently is being sent to an /address link.